### PR TITLE
GEOPY-1638: Convert mesh_type uuid to class type in ObjectForm

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-1638 - Convert mesh_type uuid to class type in ObjectForm**

### CODE REVIEW

File: geoh5py/shared/validators.py [hunk 1]
```diff
@@ -41,6 +41,42 @@
 from geoh5py.shared.utils import iterable
 
 
+def to_list(value):
```
Comment (domfournier) [inline geoh5py/shared/validators.py:44]: Can you type those?

**Review by domfournier — COMMENTED** [review]
Looks great. Can you just type the in/out of the new functions?

File: geoh5py/shared/validators.py (hunk 1 above)
Comment (benk-mira) [reply geoh5py/shared/validators.py]: done.

**Review by domfournier — APPROVED** [review]
Looks great.

**Pull request merged**